core/deployment/banned-signatures.txt

```diff
@@ -1,8 +0,0 @@
-@defaultMessage Don't use Maven classes. They won't be available when using Gradle.
```

Comment:
this seems to be pure redundancy as already in common.

config can be merged into `banned-signatures-common.txt`:


<img width="1685" alt="image" src="https://github.com/user-attachments/assets/3534277e-c4d8-4ade-a921-d1e095138c8a" />